bin/pre-commit

```diff
@@ -11,4 +11,4 @@ fi
 
 find . \( -name '*.php' \) -exec php -lf {} \;
 
-phpunit --filter WordPress "$phpcs_root/tests/AllTests.php"
+PHPCS_DIR=$phpcs_root phpunit --bootstrap="./Test/phpcs3-bootstrap.php" --filter WordPress "$phpcs_root/tests/AllTests.php"
```

Comment:
I wasn't actually using PHPCS via Composer. I had cloned it from GitHub. So I was confused.